docs/metrics.md

```diff
@@ -233,6 +233,20 @@ A counter of the number of API requests sent to the Kubernetes API.
 | `verb`        | The verb of the request, such as `Get` or `List`                   |
 | `status_code` | The HTTP status code of the response                               |
 
+This metric is calculable from `k8s_request_duration`, and it is suggested you just collect that metric instead.
```

Comment:
Should this say that it is deprecated? I feel like that would be very relevant info to be above the table instead of below

```suggestion
_Deprecated_: Use `k8s_request_duration` instead.
```